code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -83,6 +83,12 @@
 	. = ..()
 	if(mode)
 		. += "<span class='notice'>This canister is [mode]. A sticker on its side says <b>MAX PRESSURE: [siunit(pressure_limit, "Pa", 0)]</b>.</span>"
+	if(!in_range(src, user) && !isobserver(user))
+		. += "<span class='notice'>If you want any more information you'll need to get closer.</span>"
+		return
+	. += "<span class='notice'>The pressure and temperature gauges read:</span>"
```

Comment:
Maybe 
```suggestion
	. += "<span class='notice'>The two gauges on its side read:</span>"
```
Not sure on which of these is better to be honest with you, something about the pressure and temperature gauges sounds kind of awkward, not quite sure why.